site/tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    `./src/pages/**/*.{js,jsx,ts,tsx}`,
+    `./src/components/**/*.{js,jsx,ts,tsx}`,
+  ],
+  theme: {
+    extend: {},
+    fontFamily: {
+      'openSans': ['Open Sans', 'sans-serif'],
```

Comment:
Just confirming: Qanelas Soft is included and working, right?

Reply:
No, currently only 'Open Sans', i just need to add that file in project, then it will work!

Reply:
Update -> It's now added and working too in recent push